simpleflow/job/k8s.py

```diff
@@ -0,0 +1,73 @@
+from base64 import b64encode
+from jinja2 import Template
+import json
+import os
+import yaml
+
+import kubernetes.config
+import kubernetes.client
+
+from simpleflow.utils import json_dumps
+
+
+class KubernetesJob(object):
+    def __init__(self, job_name, domain, response):
+        self.job_name = job_name
+        self.response = response
+        self.domain = domain
+
+    def load_config(self):
+        """
+        Load config in the current Kubernetes cluster, either via in cluster config
+        or via the local kube config if on a development machine.
+        """
+        try:
+            kubernetes.config.load_incluster_config()
+        except:
+            kubernetes.config.load_kube_config()
+
+    def compute_job_definition(self):
+        """
+        Compute a job definition from the SWF response
+        """
+        # extract job template location
+        input = self.response.get("input")
+        if not input:
+            raise ValueError("Cannot extract job template from empty input")
+        meta = json.loads(input).get("meta", None)
```

Comment:
(minor) why `, None` here (and not above)?

Reply:
In a previous version it was a `{}` and I changed it without thinking much. Will change, better be homogeneous.